spec/controllers/transcriptions_controller_spec.rb

```diff
@@ -108,6 +108,17 @@
   describe '#show' do
     let!(:transcription) { create(:transcription) }
 
+    it 'serializes the expected attributes' do
+      get :show, params: { id: transcription.id }
```

Comment:
Not blocking: this is really a serializer unit test, longer term and adding more behaviour to serializer you should be unit testing them. 

We tested like this in panoptes and the controller specs are pretty heavy and time consuming, we should have extracted these to unit test the serializer.

Reply:
Aha! well I might as well start the precedence of testing this sort of thing within a serializer spec right from the get-go.